charts/azure-service-operator/crds/apiextensions.k8s.io_v1beta1_customresourcedefinition_azuresqldatabases.azure.microsoft.com.yaml

```diff
@@ -32,8 +31,7 @@ spec:
     singular: azuresqldatabase
   preserveUnknownFields: false
   scope: Namespaced
-  subresources:
-    status: {}
+  subresources: {}
   version: v1alpha1
```

Comment:
Removed `status` ?

Reply:
It seems like this is an `omitempty` that's been added somewhere in the toolchain. (It's in the original PR too.)